packaging/packager_pip.sh

```diff
@@ -11,60 +11,65 @@ readonly python_version="${PYTHON_VERSION:-3.8}"
 alternatives --install /usr/bin/python3 python3 /usr/bin/python${python_version} 60 \
              --slave /usr/bin/pip3 pip3 /usr/bin/pip${python_version}
 alternatives --set python3 /usr/bin/python${python_version}
-# pip dependencies in order to build some PyPI packages
+echo "-------------  pip3 dependencies (for dependencies not available as RPM) -------------"
 dnf -y install gcc libX11-devel libXtst-devel libpng-devel redhat-rpm-config
-# text matching
+echo "------------- text matching -------------"
 dnf -y install tesseract tesseract-devel
 dnf -y install gcc-c++
-# screen controlling
+echo "------------- display controlling -------------"
 dnf -y install xdotool xwd ImageMagick
-# TODO: PyAutoGUI's scrot dependencies are broken on CentOS/Rocky, currently provided offline
-#dnf -y install scrot
 dnf -y install x11vnc
-
-# dependencies that could be installed using pip
+echo "------------- upgrade pip3 and install dependencies from pip -------------"
 pip3 install --upgrade pip
-pip3 install -r "$distro_root/guibot/packaging/pip_requirements.txt"
-
-# pip packaging and installing of current guibot source
+pip3 install -r "guibot/packaging/pip_requirements.txt"
+echo "------------- pip3 packaging and installing of current guibot source -------------"
 pip3 install wheel twine
-cd "$distro_root/guibot/packaging"
+cd guibot/packaging
 python3 setup.py sdist bdist_wheel
 pip3 install dist/guibot*.whl
 
-# virtual display
+echo "------------- virtual display -------------"
 dnf install -y xorg-x11-server-Xvfb vim-common
 export DISPLAY=:99.0
 Xvfb :99 -screen 0 1024x768x24 &> /tmp/xvfb.log  &
 touch /root/.Xauthority
 xauth add ${HOST}:99 . $(xxd -l 16 -p /dev/urandom)
 sleep 3  # give xvfb some time to start
 
-# unit tests
+echo " -------- set tesseract data environment variable -------- "
+export TESSDATA_PREFIX="/usr/share/tesseract/tessdata/"
+echo "Tesseract data prefix: $TESSDATA_PREFIX"
+
+echo "------------ dump for environment variables and python path ------------"
+printenv
+python3 -c "import sys; print(sys.prefix)"
+
+echo "------------- unit tests -------------"
+pip install PyQt6
+# We need xcb lib utils to run the pyqt6 app
+dnf install -y xcb-util*
+export QT_QPA_PLATFORM_PLUGIN_PATH=/usr/local/lib/python"$python_version"/site-packages/PyQt6/Qt6/plugins/platforms
 # the tests and misc data are not included in the PIP package
-cp -r "$distro_root/guibot/tests" /usr/local/lib/python3*/site-packages/guibot/
-cp -r "$distro_root/guibot/misc" /usr/local/lib/python3*/site-packages/guibot/
-cd /usr/local/lib/python3*/site-packages/guibot/tests
-if (( distro_version <= 7 )); then
-    COVERAGE="python3-coverage"
-else
-    COVERAGE="coverage-${python_version}"
-fi
+cp -r "../tests" /usr/local/lib/python3*/site-packages/guibot/
+cp -r "../misc" /usr/local/lib/python3*/site-packages/guibot/
+cd /usr/local/lib/python"$python_version"/site-packages/guibot/tests/
```

Comment:
Why are some of these version using a wildcard while others use an env variable? How about just sticking to the env variable for all of them?